README.rst

```diff
@@ -94,6 +94,19 @@ or
 
 $ python host_os.py build-package --help
 
+* Build old versions
+
+OpenPOWER Host OS 1.0 and 1.5 used an older CentOS (7.2) as base version. 
+To build it, you must do the following changes in this git repository
+before building:
+ * Replace "http://mirror.centos.org/altarch/7" with "http://vault.centos.org/altarch/7.2.1511/"
+   in mock_configs/CentOS/7/CentOS-7-ppc64le.cfg
+ * Replace distro_version value in config.yaml from "7" to "7.2"
+ * Replace "mock_config/CentOS/7" child subnode in config.yaml from "7" to "7.2"
+ * Replace build_version in config.yaml to v1.5.0 or v1.0.0
```

Comment:
OK for me. I just prefer to use 2 more explicit seds then the one you use to update the distro version.